tests/test_shorthand.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List, Optional
+
+import pytest
+
+from streaming.util.shorthand import get_list_arg, normalize_bytes, normalize_count
+
+
+@pytest.mark.parametrize(('text', 'expected_output'), [('hello,world', ['hello', 'world']),
+                                                       ('hello', ['hello']), ('', [])])
+def test_get_list_arg(text: str, expected_output: List[Optional[str]]):
+    output = get_list_arg(text)
+    assert output == expected_output
+
+
+def test_normalize_bytes():
+    input_to_expected = [
+        ('1234', 1234),
+        ('1b', 1),
```

Comment:
Does an input like '1.5e9mib' work?

Reply:
Nope, it does not take scientific notation amounts of units, as units and e# are doing kind of the same thing...

```
>>> from streaming.util.shorthand import normalize_bytes
>>> normalize_bytes('1.5e9mib')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ape/dev/STREAMING/arg-shorthand/streaming/util/shorthand.py", line 335, in normalize_bytes
    raise ValueError(f'Invalid bytes: {size}. Reasons: {[str(e) for e in errors]}.')
ValueError: Invalid bytes: 1.5e9mib. Reasons: ['Unit is unknown: e in part 0 (input: 1.5e9mib).', 'Unit is unknown: e in part 0 (input: 1.5e9mib).'].
```

Reply:
And scientific notation by itself in your yaml is going to get floated